db/migrate/20160905034718_change_project_steps_to_timeline_entries.rb

```diff
@@ -0,0 +1,11 @@
+class ChangeProjectStepsToTimelineEntries < ActiveRecord::Migration
+  def change
+    rename_table  :project_steps, :timeline_entries
+    rename_column :project_logs,  :project_step_id, :timeline_entry_id
+
+    add_column :timeline_entries, :type, :string
```

Comment:
I think there should be an index on this, and a non null constraint. I am generally a fan of DB constraints wherever they are applicable.